robustifier/src/main/kotlin/cmu/isr/ts/nfa/NFAParallelComposition.kt

```diff
@@ -61,6 +61,23 @@ open class NFAParallelComposition<S1, S2, I>(
     }
   }
 
+  /**
+   * I'm ignoring the input, sorry not sorry
+   */
+  override fun getStates(input: MutableIterable<I>?): MutableSet<Pair<S1, S2>> {
+    val states = mutableSetOf<Pair<S1,S2>>()
```

Comment:
I would recommend a better way to do it. In your tolerance package, e.g., Utils.kt. You can define
```
fun <S1, S2, I> NFAParallelComposition<S1, S2, I>.getAllStates(): MutableSet<Pair<S1, S2>> {
...
}
```
In this way, you won't need to change my original implementation. Also, you don't have to override the getStates(input) method.

Reply:
I can certainly do this (and the suggestion below) instead. Do you suggest this because you think my additions will break your existing tools?